.jenkins/nightly.groovy

```diff
@@ -8,10 +8,10 @@ pipeline {
 
         stage('Build') {
             parallel {
-                stage('CUDA-11.4.2') {
+                stage('CUDA-11.7.1') {
                     agent {
                         docker {
-                            image 'nvidia/cuda:11.4.2-devel-ubuntu20.04'
+                            image 'nvidia/cuda:11.7.1-devel-ubuntu22.04'
```

Comment:
Why `11.7.1`?
For the record, looking at docker hub https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=ubuntu22, we can chose from [`11.7.0`, `11.7.1`, `11.8.0`, `12.0.0`]

Reply:
Well, out of those versions, I wanted the earlierst, but thought that maybe `11.7.1` fixed some bugs in `11.7.0`.